src/helpers/Utils/Utils.test.js

```diff
@@ -345,4 +349,31 @@ describe('Utils tests', () => {
       ).toBe(`${isoDate}Z`);
     });
   });
+
+  describe('parseDateTime', () => {
+    it('replaces the position of an element into an array immutable-ish', () => {
+      const array = ['a', 'b', 'c'];
+      deepFreeze(array);
+      const result = replaceItemOfArray(array, 2, 'v');
+      expect(result).toEqual(['a', 'b', 'v']);
+    });
+  });
+
+  describe('parseDateTime', () => {
+    it('removes an element from the array immutable-ish', () => {
+      const array = ['a', 'b', 'c'];
+      deepFreeze(array);
+      const result = removeFromArray(array, 2);
+      expect(result).toEqual(['a', 'b']);
+    });
+  });
+
+  describe('parseDateTime', () => {
+    it('reorders an array immutable-ish', () => {
+      const array = ['a', 'b', 'c'];
+      deepFreeze(array);
+      const result = reorderArray(array, 2, 0);
+      expect(result).toEqual(['c', 'a', 'b']);
+    });
+  });
```

Comment:
@robgietema this claims to be describing `parseDateTime`!

Reply:
Good catch, fixed